3p/integration.js

```diff
@@ -273,6 +273,7 @@ const AMP_EMBED_ALLOWED = {
   dable: true,
   engageya: true,
   epeex: true,
+  idealmedia: true,
```

Comment:
Can I ask the reason to support `<amp-embed>`. This change migrate the `<amp-ad>` to using `<amp-embed>` instead. Thanks.

Reply:
we want to give our publishers the opportunity to integrate amp

Reply:
To help me understand they should already be able to integrate with `<amp-ad>`. What is the reason here to use `<amp-embed>`? Thanks.

Reply:
Thanks @zhouyx for your question 
   Our network has 2 different types of cooperation with partners 
1) Advertisement. We are showing ad using our widgets. 
For such cooperation we'd like to use <amp-ad>
2) News exchange. Such type of cooperation doesn't have any monetization so would like to use <amp-embed>